web_console/capabilities_products-web-console.adoc

```diff
@@ -1,53 +1,36 @@
-* xref:../web_console/web_terminal/installing-web-terminal.adoc#installing-web-terminal[Installing the web terminal]
-
 //OpenShift LightSpeed
 include::modules/openshift-lightspeed-web-console.adoc[leveloffset=+1]
-
-[role="_additional-resources"]
-.Additional resources
-* link:https://docs.redhat.com/en/documentation/red_hat_openshift_lightspeed/1.0tp1/html/about/ols-about-openshift-lightspeed#ols-openshift-lightspeed-overview[{ols} overview]
-* link:https://docs.redhat.com/en/documentation/red_hat_openshift_lightspeed/1.0tp1/html/install/ols-installing-lightspeed[Installing {ols}]
-
 //pipelines
 include::modules/pipelines-web-console.adoc[leveloffset=+1]
-
-[role="_additional-resources"]
-.Additional resources
-* link:https://docs.openshift.com/pipelines/1.14/create/working-with-pipelines-web-console.html[Working with {pipelines-title} in the web console]
-* link:https://docs.openshift.com/pipelines/1.14/create/working-with-pipelines-web-console.html#op-console-statistics_working-with-pipelines-web-console[Pipeline execution statistics in the web console]
-
 //serverless
 include::modules/serverless-web-console.adoc[leveloffset=+1]
-
-[role="_additional-resources"]
-.Additional resources
-* link:https://access.redhat.com/documentation/en-us/red_hat_openshift_serverless/1.31/html/installing_serverless/install-serverless-operator#serverless-install-web-console_install-serverless-operator[Installing the {ServerlessProductName} Operator from the web console].
-
 //RHDH
 include::modules/rhdh-web-console.adoc[leveloffset=+1]
-
 include::modules/rhdh-install-web-console.adoc[leveloffset=+2]
 
 [role="_additional-resources"]
 .Additional resources
+* xref:../operators/understanding/olm-understanding-software-catalog.adoc#olm-understanding-software-catalog[Understanding the software catalog]
+* xref:../web_console/web_terminal/installing-web-terminal.adoc#installing-web-terminal[Installing the web terminal]
+* link:https://docs.redhat.com/en/documentation/red_hat_openshift_lightspeed/1.0tp1/html/about/ols-about-openshift-lightspeed#ols-openshift-lightspeed-overview[{ols} overview]
+* link:https://docs.redhat.com/en/documentation/red_hat_openshift_lightspeed/1.0tp1/html/install/ols-installing-lightspeed[Installing {ols}]
+* link:https://docs.openshift.com/pipelines/1.14/create/working-with-pipelines-web-console.html[Working with {pipelines-title} in the web console]
+* link:https://docs.openshift.com/pipelines/1.14/create/working-with-pipelines-web-console.html#op-console-statistics_working-with-pipelines-web-console[Pipeline execution statistics in the web console]
+* link:https://access.redhat.com/documentation/en-us/red_hat_openshift_serverless/1.31/html/installing_serverless/install-serverless-operator#serverless-install-web-console_install-serverless-operator[Installing the {ServerlessProductName} Operator from the web console]
```

Comment:
Are we allowed only one Additional References per assembly?

Reply:
There's a warning that flagged separated Additional Resources. This may be a malleable rule.